guides/common/modules/proc_importing-ansible-roles.adoc

```diff
@@ -2,11 +2,12 @@
 
 = Importing Ansible Roles and Variables
 
-You can import Ansible roles and variables from the `/etc/ansible/roles` directory on {Project} or on a {SmartProxy} that has Ansible enabled.
+You can import Ansible roles and variables from the `/etc/ansible/roles` directory on {ProjectServer} or {SmartProxy} that has Ansible enabled.
```

Comment:
we also import roles and variables from `/usr/share/ansible/roles` and also from collections installed at  `/etc/ansible/collections` and at `/usr/share/ansible/collections`. These paths are based on the configuration present at /etc/ansible/ansible.cfg (roles_path and collections_path directives). If none are specified, the values I listed earlier are used. The same paths are expected to be set at /etc/foreman-proxy/ansible.cfg (this cfg file is used when the ansible run is triggered). So importing uses one config file, while running imported roles uses second.